Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:jessie
```

Comment:
True. I wonder though how would behave dbus inside the docker and other low level system communication libs.
I believe using an Alpine for docker is the prefered way to go considering the duplication of systems though.
I'm not totally sure in this case because of the low layers libs and the Kiwix dependency.
Opinions welcome.